covid19-etl/etl/ncbi.py

```diff
@@ -121,7 +148,9 @@ def read_ncbi_manifest(bucket, key, accession_number_filename_map):
     line_stream = codecs.getreader("utf-8")
     for line in line_stream(s3_object.get()["Body"]):
         words = line.split("\t")
-        accession_number_filename_map[words[1]] = words[5].split("/")[-1]
+        r1 = re.findall("[SDE]RR\d+", words[5])
```

Comment:
is this sort of a validation of the accession number syntax? 🤔

Reply:
yes. @wangfan860 told me that an accession number is always in the format of `SRRxxxx`, `DRRxxxx` or `ERRxxxx`

Reply:
yes, to expand on this a bit, the accession numbers were determined by which database the sample is submitted to. SRR for data submitted to NCBI, ERR for EMBL-EBI (European Molecular Biology Laboratory), and DRR for DDBJ (DNA Data Bank of Japan).

Reply:
the reference for the above statement😊 https://www.ncbi.nlm.nih.gov/books/NBK56913/

Reply:
so we're assuming that data will never come from different sources? 🤔 idk, i think maybe we should just accept all accession numbers?

Reply:
I think so, but let me double check with NCBI people.

Reply:
We need to know the format of accession number so that we can extract it correctly from the manifest. The manifest does not have any indication to say something like `this is the accession number`. Since the code is specifically for NCBI data so I think it is fine to keep that format assumption

Reply:
ah got it

Reply:
then can we please add the explanation from Fan either in the code or in the readme